adaptio-shelves/manifest.gen.ts

```diff
@@ -0,0 +1,12 @@
+// DO NOT EDIT. This file is generated by deco.
```

Comment:
What's this adaptio-shelves app for?

Reply:
@tlgimenes Sorry, was using our repository as a solution to hackathon, now it's fixed!